src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1068,13 +1112,11 @@ class ImGuiConsole
 		ImGui::SetCursorPosY(ImGui::GetCursorPosY() + 4);
 		ImGui::PushItemWidth(ImGui::GetContentRegionAvailWidth());
 		ImGui::PushStyleColor(ImGuiCol_FrameBg, ImVec4(0.00f, 0.00f, 0.00f, 0.00f));
-		ImGui::PushFont(mq::imgui::ConsoleFont);
```

Comment:
As mentioned above, you've got a logic error in your understanding of how the fonts are being used. We use a variable-width font for all the UI and a monospace font for the console. With your changes, everything is now using the monospace font which is not ok.

If you want to change fonts, you need to change the console font.